packages/accounts/src/__tests__/__setup__.ts

```diff
@@ -1,16 +1,15 @@
 import type { Address } from '@solana/addresses';
 import type { Decoder } from '@solana/codecs-core';
-import type { PendingRpcRequest, Rpc, RpcResponse } from '@solana/rpc-types';
-
+import type { PendingRpcRequest, Rpc } from '@solana/rpc-spec';
 import type {
     AccountInfoBase,
     AccountInfoWithBase58Bytes,
     AccountInfoWithBase58EncodedData,
     AccountInfoWithBase64EncodedData,
-    GetAccountInfoApi,
-    GetMultipleAccountsApi,
-    JsonParsedDataResponse,
-} from '../rpc-api';
+    SolanaRpcResponse,
+} from '@solana/rpc-types';
+
+import type { GetAccountInfoApi, GetMultipleAccountsApi, JsonParsedDataResponse } from '../rpc-api';
```

Comment:
Remind me again why we have to re-implement these in this package? It seems like through your refactoring, `@solana/rpc` doesn't need `@solana/accounts`, neither does `@solana/rpc-api`.

Reply:
Yeahhhh I think we should hold off on that. That's like 60 packages! 🫠

Reply:
Agreed! 😅